data/meta/Game.lua

```diff
@@ -0,0 +1,101 @@
+-- Copyright © 2008-2022 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+-- This file implements type information about C++ classes for Lua static analysis
+-- TODO: this file is partially type-complete, please expand it as more types are added.
+
+---@meta
+
+---@class Game
+---
+---@field player Body #TODO: add Ship / Player to type tree
+---@field system StarSystem? The system the game is currently playing in or nil when in hyperspace.
+---@field systemView unknown #TODO: add type info for SystemView interface
```

Comment:
On the word `unknown`:
`[Lua Diagnostics. undefined-doc-name] [W] Undefined type or alias.`